ballerina/stream_types.bal

```diff
@@ -0,0 +1,555 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+public class TopicStream {
```

Comment:
Shall we add API docs to the stream object? Do we need to make them public?

Reply:
[6e6ea03](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/6e6ea032e00829a63a07d06848d40cdd66c011c0)